### ISSUE TITLE
Game freezes when joining a server

### ISSUE THREAD

**Issue by AngryCarrot789** [xJon/The-1.12.2-Pack#903]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
**yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Information required**
1. Is the modpack crashing: **no**
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): **link to paste**
3. Is Optifine installed or any other additional mods: **no**

**Describe the issue**

When I join a server, the game just freezes for a long time. I managed to get a VisualVM screenshot of what it's waiting on, hopefully it helps:
![image](https://github.com/xJon/The-1.12.2-Pack/assets/32274404/d74d0927-3089-4fb7-b193-c3596c50fd83)

My last logout location was next a bunch of Computronics speakers so maybe it could also be something to do with that?

**Comment by xJon** [comment]
Seems to be a duplicate of https://github.com/xJon/The-1.12.2-Pack/issues/876, see https://github.com/xJon/The-1.12.2-Pack/issues/876#issuecomment-1512895802 for a workaround, and an update fixing this should be released very soon :)
